lib/src/com/mapswithme/maps/api/MWMPoint.java

```diff
@@ -63,10 +74,20 @@ public MWMPoint(double lat, double lon, String name, String id)
    */
   public void setId(String id) { mId = id; }
 
+  /**
+   * Sets the style (appearance) for this point. Null, empty-string or unrecognized appears as a
+   * violet circle. Styles known to MAPS.ME appear with the appropriate symbol. At the time of
+   * writing, supported styles are "placemark-red", "placemark-blue", "placemark-purple",
+   * "placemark-yellow", "placemark-pink", "placemark-brown", "placemark-green", "placemark-orange",
+   * all of which are displayed as a small flag of the indicated colour.
+   * @param style
+   */
+  public void setStyle(String style) { mId = style; }
+
   @Override
   public String toString()
   {
-    return "MWMPoint [lat=" + mLat + ", lon=" + mLon + ", name=" + mName + ", id=" + mId + "]";
+    return "MWMPoint [lat=" + mLat + ", lon=" + mLon + ", name=" + mName + ", id=" + mId + ", style=" + mStyle + "]";
```

Comment:
It is a matter of one's taste, but making this line multi-line will make code a bit more readable.